dietstory-routing/login_bonus/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from . import views
+from .initialize_rewards import populate_login_rewards
+
+populate_login_rewards()
```

Comment:
At the moment, I only know the urls file is best place to load the rewards, but the apps.py for the app may work. When I tested with the settings file, it the apps are not loaded yet, so threw an error.

Reply:
Now that I think about it, init file will probably work. Will try tonight

Reply:
init file did not solve problem because apps would not be loaded in time. Decided to move ahead with writing a custom command instead of using fixture files. This needs a change to the run_server docker image though. That is we need to add python -u manage.py initialize_rewards  to the list of commands.

Reply:
We initialize through another post request that I developed by passing a csv file.